### PR TITLE
results styling

### PR DESCRIPTION
**Pull request by mahmedov**

### CODE REVIEW

**Comment by mahmedov** [comment]
It's not much but I can work more on it, this is a foundation per se.

File: src/components/Results.jsx [hunk 1]
```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { connect } from 'react-redux';
 
 const ResultsContainer = props => (
-  <div className='voting-result'>
-    <h4 className='voting-title'>What's better?</h4>
+  <div className='voting-result vr'>
```
Comment (eugrdn) [inline src/components/Results.jsx]: why don't use existed class?

File: src/index.css [hunk 2]
```diff
@@ -53,6 +62,10 @@ table {
 html, body, {
   min-height: 100%;
 }
+div {
+  text-align: center;
+  display: block;
```
Comment (eugrdn) [inline src/index.css]: I think this is redundant

**Comment by eugrdn** [comment]
@mahmedov, good 👍  it's enough, but if you feeling the inspiration, you could work as much as you want 😄

File: src/index.css [hunk 3]
```diff
@@ -21,14 +22,22 @@ time, mark, audio, video {
...
-article, aside, details, figcaption, figure, 
+article, aside, details, figcaption, figure,
 footer, header, hgroup, menu, nav, section {
   display: block;
 }
 body {
   line-height: 1;
   padding: 0;
   margin: 0;
+  width: 100wh;
```
Comment (eugrdn) [inline src/index.css]: wh?

File: src/index.css [hunk 4]
```diff
@@ -21,14 +22,22 @@ time, mark, audio, video {
...
+  height: 90vh;
```
Comment (eugrdn) [inline src/index.css]: why 90?

File: src/index.css (hunk 3 above)
Comment (mahmedov) [reply src/index.css]: typo

File: src/components/Results.jsx (hunk 1 above)
Comment (mahmedov) [reply src/components/Results.jsx]: I was testing it out and forgot to remove it

File: src/index.css (hunk 3 above)
Comment (mahmedov) [reply src/index.css]: typo

**Pull request closed without merging**